### ISSUE TITLE
Disabling tiling in config file

### ISSUE THREAD

**Issue by jackflint96** [hyprwm/Hyprland#1721]
It would be amazing if there was a way for disabling tiling from the config file

**Comment by vaxerski** [comment]
> Tiling Wayland Compositor
> Hey how 2 disable tiling?????

**Comment by tchofy** [comment]
Agreed with the bizarre question, but...you can just use a window rule for that.
windowrulev2=float,class:.*
